package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@openmrs/esm-login-app",
   "version": "3.0.1",
   "description": "The login microfrontend for the OpenMRS SPA",
-  "main": "dist/openmrs-esm-login.js",
+  "main": "dist/openmrs-esm-login-app.js",
```

Comment:
Why is this changed? I also don't see a change in the travis file.

If we change something as fundamental as this, then we need to make sure to be 100% consistent.

Reply:
Makes the devtools port-to-URL interpolation work. Good catch about Travis! And yes, I agree we should be consistent about it.